rqd/rqd/cuerqd.py

```diff
@@ -54,7 +27,7 @@
 from builtins import object
 import os
 import sys
-import getopt
+import argparse
```

Comment:
nit: Alphabetize the `import` lines. So `import argparse` should be above the `from builtins` lines.